bed/methods.go

```diff
@@ -46,24 +46,27 @@ func (b *BedSlice) Pop() interface{} {
 }
 
 func (b BedSlice) Write(file string) {
-	Write(file, b, 7)
+	a := make([]Bed, len(b))
```

Comment:
It catches my eye that we are making a copy of the slice here, which could be large.  Is it worth maintaining a WritePtr as well as a Write to avoid this copy?  I am thinking that BedSlice is used by the overlap program?